README.md

```diff
@@ -30,6 +30,23 @@ The gradebook builds on the awesome work of @jwo and @matthiasak. Each student y
 
 Data is cached in localStorage so as not to DDoS TIYO (too much). Clicking the "Refresh Grades" button will scrape the site once again and store the new data.
 
+### Homework
+
+The Homework module builds upon the inspiration from @jah2488.  It allows communication to an authorized ruby process running on your computer to rapidly clone and setup git based projects on your computer.  This allows for easy cloning of PR's, Git Repos, Gists, and various other git related links.  For full documentation please read more about the underlying tool newline_hw [here](https://github.com/TIYDC/newline_hw#newlinehw).
+
```

Comment:
I understand not wanting to document in 3 places, but I don't see anywhere where you tell people what to actually do other than install. In other words, if I'm on my path and click on the download icon I do not see a "clone hw" button. I know why - but someone else might not. If you want to document in that dropdown that's cool, but I don't see that. :/